tools/check_release.py

```diff
@@ -137,19 +160,21 @@ def invoke_api(payload, url, auth, polls, begin="start/"):
         time.sleep(poll_delay)
         r = requests.get(url + "output/%s" % uuid, auth=auth)
         response = r.json()
-        if response['result']['data']['task_complete']:
-            
+
+        data = response['result']['data']
```

Comment:
Are we garenteed to have `response['result']['data']` here? it might be worth a `try:` `except KeyError:` or similar. Not blocking, but maybe we should consider it...